### PR TITLE
feat: Add medium and Discourse links to Across.to

### PR DESCRIPTION
**Pull request by Tulun**
Story details: https://app.shortcut.com/uma-project/story/4346

Note: Missing urls, this is for design QA once I get those.
Screen Shot 2022-02-09 at 1 40 12 PM

### CODE REVIEW

File: src/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -22,6 +24,16 @@ const NAV_LINKS = [
...
     icon: GithubLogo,
   },
+  {
+    name: "Medium",
+    url: "",
+    icon: MediumLogo,
+  },
+  {
+    name: "Discourse",
+    url: "",
```
Comment (daywiss) [inline src/components/Layout/Layout.tsx:34]: i guess we are waiting for the links

Comment (Tulun) [reply src/components/Layout/Layout.tsx:34]: Yeah asked Tom for them but wanted the PR open for design QA (as this is a trivial change)

**Review by Tommy1231232 — CHANGES_REQUESTED** [review]
Feedback from Jesper:
Screen Shot 2022-02-16 at 7 43 51 AM

**Review by daywiss — APPROVED** [review]
lgtm, can probably get rid of some of the commented out props

**Review by Tommy1231232 — CHANGES_REQUESTED** [review]
@Tulun colors need to be updated to match designs: 

Screen Shot 2022-02-18 at 2 00 41 PM

**Comment by Tulun** [comment]
> @Tulun colors need to be updated to match designs:
> 
> Screen Shot 2022-02-18 at 2 00 41 PM

Can you confirm with Jesper? these are the svg logos he sent me. I need either new ones for the old icons or the new ones. Thanks.

**Comment by Tommy1231232** [comment]
@Tulun - the format of the input fields is shifted from the tabs at the top. They should be aligned. The icons at the botton still seem too light as well but will have Jesper comment on them as he knows better. 


Screen Shot 2022-02-25 at 5 14 41 PM

**Review by Tommy1231232 — APPROVED** [review]

**Pull request merged**